src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,10 +1150,10 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
-    }
 
+    private static String removePrefix(String fullstirng, String prefix) {
+        return fullstirng.replace(prefix, "");
+    }
     /**
```

Comment:
Do not forget to update the documentation of the function as well. You can look into eclipse renaming shortcuts which will do it automatically for you.